### PR TITLE
[KYV-700] Added new merchant configuration key merchantShopId.

### PR DESCRIPTION
**Pull request by martinldev**
Added new merchant configuration key merchantShopId.

### CODE REVIEW

File: merchantapi/src/test/java/fi/hel/verkkokauppa/configuration/api/merchant/MerchantControllerTest.java [hunk 1]
```diff
@@ -79,7 +79,7 @@ public void returnsError404IfMerchantIsNotFoundTest() throws Exception {
         assertEquals(CommonApiException.class, exception.getClass());
         assertEquals(HttpStatus.NOT_FOUND, exception.getStatus());
         assertEquals("merchant-not-found", exception.getErrors().getErrors().get(0).getCode());
-        assertEquals("merchant with id [merchantId] not found", exception.getErrors().getErrors().get(0).getMessage());
+        assertEquals("merchant with value: [merchantId] not found", exception.getErrors().getErrors().get(0).getMessage());
```
Comment (severikupari1) [inline merchantapi/src/test/java/fi/hel/verkkokauppa/configuration/api/merchant/MerchantControllerTest.java:82]: Why did you change this to value?

Comment (martinldev) [reply merchantapi/src/test/java/fi/hel/verkkokauppa/configuration/api/merchant/MerchantControllerTest.java:82]: The error message was with "value:" not "id", so I changed it. The Test did not pass with the "id" text.
